packages/backend-defaults/config.d.ts

```diff
@@ -1164,5 +1164,15 @@ export interface Config {
        */
       plugins: string[];
     }>;
+
+    /**
+     * A list of deployed Backstage instances that can be crawled for discovery.
```

Comment:
Not immediately obvious to me: is this supposed to be individual instances, i.e. pods? or service name pointing to multiple instances of the same class, i.e. deployments. That question also got me thinking about whether there's value in supporting custom DNS lookups so that we can resolve all A/AAAA/SRV records and reach each individual instance for metadata collection. For the purpose of detecting if a feature is supported I would for example expect that all pods in a deployment report that they support the feature.